src/modules/settings/MultiSetting.ts

```diff
@@ -0,0 +1,30 @@
+import BaseSetting from './BaseSetting';
+
+export default class MultiSetting<T> extends BaseSetting<T[], T> {
+    private optionValues;
+
+    validValue(value: T[]): boolean {
+        if (!this.isUnlocked(value)) {
+            return false;
+        }
+
+        if (value.length === 0) {
+            return true;
+        }
+        if (this.options.length === 0) {
+            return false;
+        }
```

Comment:
These checks don't seem needed, both get covered by the "are all values included in optionValues" bit

Reply:
Removed